modules/solid_mechanics/src/materials/ADComputeIncrementalShellStrain.C

```diff
@@ -353,6 +353,7 @@ ADComputeIncrementalShellStrain::computeGMatrix()
         gmn(0, 2) += (*_dxyz_dxi[j])[i](component) * (*_dxyz_dzeta[j])[i](component);
         gmn(1, 2) += (*_dxyz_deta[j])[i](component) * (*_dxyz_dzeta[j])[i](component);
 
+        // why are we dropping derivatives here?!
```

Comment:
There are a bunch of these that don't actually have derivatives, and don't need to be AD properties. That's separate from this, though.